httpclient/src/integration-tests/e2e.kt

```diff
@@ -0,0 +1,141 @@
+//import tbdex.sdk.httpclient.RealTbdexClient
```

Comment:
integration-tests/e2e.kt does not have the little ⏯️ button that shows up on intellij. but test/kotlin/tbdex/sdk/httpclient/e2e.kt seems to let me play it. 

welcome to suggestions on where this e2e should actually live

Reply:
It's not runnable because intellij doesn't know it's a source dir. You can set that: 
<img width="463" alt="image" src="https://github.com/TBD54566975/tbdex-kt/assets/1435885/fab40b1f-3e48-4588-a9bc-6912ae136f68">

Reply:
just yeeted the whole thing